dev/devicelab/lib/framework/browser.dart

```diff
@@ -232,15 +230,12 @@ String _findSystemChromeExecutable() {
     return '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
   } else if (io.Platform.isWindows) {
     const String kWindowsExecutable = r'Google\Chrome\Application\chrome.exe';
-    final List<String> kWindowsPrefixes = <String>[
+    final List<String> kWindowsPrefixes = <String?>[
       io.Platform.environment['LOCALAPPDATA'],
       io.Platform.environment['PROGRAMFILES'],
       io.Platform.environment['PROGRAMFILES(X86)'],
-    ];
+    ].whereType<String>().toList();
```

Comment:
Ah clever way to remove nulls, I hadn't thought of that before.

Reply:
I stole it from @jmagman's review of another null safety migration.